cm-gateway/gateway-controller-install/projectcontour/contour.yaml

```diff
@@ -0,0 +1,5236 @@
+#! This file comes from https://raw.githubusercontent.com/projectcontour/contour/release-1.22/examples/render/contour.yaml
```

Comment:
Suggestion: use [Contour's Helm chart](https://artifacthub.io/packages/helm/bitnami/contour) instead of committing the yamls to this repo as that will be more consistent with how we install other addons and easier to manage. The Helm chart is not issued by Contour authors, however they have added installing using this chart to their [installation guide](https://projectcontour.io/getting-started/#option-2-helm), so I think this can be considered sufficiently mature and reliable

(I think this chart is a recent addition- we might also want to start using it to install the Contour that's used for cert-manager e2e tests https://github.com/cert-manager/cert-manager/issues/5374)

Reply:
I have tested [this helm installation](https://projectcontour.io/getting-started/#option-2-helm), but it seems not to support `Gateway Provisioner`, it lacks `clusterrole`, `clusterrolebinding` and other resources associated with `gateway`. So I select the installation using raw yaml file.

Could you help me to check if this [chart](https://artifacthub.io/packages/helm/bitnami/contour) is avaliable for our scenario?